staging/src/k8s.io/apiserver/pkg/endpoints/openapi/openapi.go

```diff
@@ -146,6 +146,15 @@ func friendlyName(name string) string {
 	return strings.Join(nameParts, ".")
 }
 
+// Grabs the last 2 components of the package name and joins them by '.'
+func friendlyNamev3(name string) string {
```

Comment:
That function could be unit tested.

Reply:
+1. This has to test all edge case, multi component group name with dots, dashes, slashed, ....

Reply:
This is tested by testing against the public API surface GetDefinitionNamev3 which was also added in this PR